frontend/src/reducers/plannerReducer.js

```diff
@@ -111,9 +113,12 @@ const plannerReducer = (state = initialState, action) => {
 
     case "TOGGLE_WARNINGS":
       let coursesCpy = new Map(state.courses);
+      console.log(coursesCpy);
```

Comment:
can remove for less console clutter, but otherwise looks good! 😎

Reply:
Oops i forgot. Thanks for pointing that out 🙏